src/bicep/examples/zeroTrustWorkbook/README.md

```diff
@@ -0,0 +1,95 @@
+
+Required Parameters | Description
+------------------- | -----------
+_location_ | The region to deploy Azure Sentinel into
+_resourcePrefix_ | A 3-10 alphanumeric character string without whitespace, used to prefix resources and generate uniqueness for resources with globally unique naming requirements
+_deploySentinel_ | A boolean expression indicating that Azure Sentinel is to be deployed with the MissionLZ deployment
+_deployASC_ | A boolean expression indicating that Azure Security Center (or Microsoft Defender for Cloud) is enabled in the MissionLZ deployment
+
+An example deployment with required deployment parameters included is shown below:
+
+```bash
+# after a Mission LZ deployment
+#
+# az deployment sub create \
+#   --subscription $deploymentSubscription \
+#   --name "myMlzDeployment" \
+#   --template-file ./mlz.bicep \
+#   --parameters resourcePrefix=$resourcePrefix \
+#   deploySentinel=true \
+#   deployASC=true
+```
+
+## Deploying Sentinel Zero Trust (TIC3.0) Workbook
+
+The Sentinel Zero Trust (TIC3.0) Workbook is maintained in the [Azure Sentinel GitHub repository](https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/ZeroTrust(TIC3.0)/readme.md)
+
+With the link provided, it is possible to use the "Deploy to Azure" button with some simple input parameters for Azure Government and Azure Commercial clouds.
+
+From the pre-existing deployment of MLZ shown above, the following parameters are required for deployment of the Azure Sentinel Workbook:
+
+ Required Input Parameters | Description
+---------------------------|------------
+_operationsSubscriptionId_ | The subscription that contains the Log Analytics Workspace and to deploy the Sentinel solution into
+_operationsResourceGroupName_ | The resource group that contains the Log Analytics Workspace to link Azure Sentinel to
+_logAnalyticsWorkspaceName_ | The name of the Log Analytics Workspace to link Azure Sentinel to
+
+One way to retreive these values is with the Azure CLI:
+
+az deployment sub show \
+  --subscription $deploymentSubscription \
+  --name "myMlzDeployment" \
+  --query properties.outputs
+
+...which should return an object containing the values you need:
+
+```plaintext
+{
+  "operationsSubscriptionId": {
+    "type": "String",
+    "value": "0987654-3210..."
+  },
+  ...
+  "operationsResourceGroupName": {
+    "type": "String",
+    "value": "mlz-dev-operations"
+  },
+  ...
+  "logAnalyticsWorkspaceName": {
+    "type": "String",
+    "value": "mlz-dev-laws"
+  },
+  ...
+```
+
+...and if you're on a BASH terminal, this command (take note to replace "myMlzDeployment" with your deployment name) will export the values as environment variables:
+
+```bash
+export $(az deployment sub show --name "myMlzDeployment" --query "properties.outputs.{ args: [ join('', ['operationsResourceGroupName=', operationsResourceGroupName.value]), join('', ['logAnalyticsWorkspaceName=', logAnalyticsWorkspaceName.value])] }.args" --out tsv | xargs)
```

Comment:
This example command does not work as there's no `operationsResourceGroupName` output at the parent level of `properties.outputs`.

My recommendation is to remove L82 to L86 entirely and let the user source the parameters however they see fit.

Out of curiosity, I referenced `spokes.value[1].resourceGroupName` to make the snippet functional:

```plaintext
export $(az deployment sub show --name "myMlzDeployment" --query "properties.outputs.{ args: [ join('', ['operationsResourceGroupName=', spokes.value[1].resourceGroupName]), join('', ['logAnalyticsWorkspaceName=', logAnalyticsWorkspaceName.value])] }.args" --out tsv | xargs)
```

Reply:
This was originally in https://github.com/Azure/missionlz/tree/main/src/bicep/examples/sentinel
Used this in place of my original instructions on gathering required data sources. 
Will update.